solara/cache.py

```diff
@@ -15,7 +15,11 @@
     overload,
 )
 
-import cachetools
+try:
+    import cachetools
+except ModuleNotFoundError:
+    cachetools = None  # type: ignore
```

Comment:
While we are at it, we could fix the typing on these imports?
```suggestion
from types import ModuleType

cachetools: Optional[ModuleType]
try:
    import cachetools
except ModuleNotFoundError:
    cachetools = None
```

Reply:
I'll start doing this from now on:
```
try:
    import cachetools
    has_cachetools = True
except ModuleNotFoundError:
    has_cachetools = False
```

And rely on the boolean